.env.template.testnet.gobi

```diff
@@ -3,8 +3,25 @@ COMPOSE_PROJECT_NAME=evmapp-testnet-gobi
 INTERNAL_NETWORK_SUBNET='10.10.50.0/24'
 SCNODE_IP_ADDRESS='10.10.50.3'
 
+ZEN_TAG=v4.1.1
+ZEN_PORT=19033
+ZEN_RPC_USER=zenuser
+ZEN_RPC_PASSWORD=password
```

Comment:
Same comment here

Reply:
Same (The zend node is not going to expose rpc ports so this looks safe and wallet should also stay empty. The users can always manually edit the configuration should they want to do something else.)